WebHostLib/api/__init__.py

```diff
@@ -21,12 +22,22 @@ def room_info(room: UUID):
     room = Room.get(id=room)
     if room is None:
         return abort(404)
+    
+    def supports_apdeltapatch(game: str):
+        return game in worlds.Files.AutoPatchRegister.patch_types
+    downloads = {}
+    for slot in sorted(room.seed.slots):
+        if slot.data and not supports_apdeltapatch(slot.game):
+            downloads[slot.player_id] = url_for("download_slot_file", room_id=room.id, player_id=slot.player_id)
```

Comment:
It should be noted that technically this makes the player_id a str key on the json object; technically the recommended format is therefore a list of objects; example
{1 : data1,  2: data2}
becomes
data1["slot"] = 1
data2["slot"] = 2
[data1, data2]

We have this violation of json spec in a few places, it might be good to avoid increasing that count and convert where the opportunity arises?